tests/model_inheritance_regress/tests.py

```diff
@@ -496,3 +496,69 @@ def test_related_filtering_query_efficiency_ticket_15844(self):
                 r.supplier_set.all(),
                 [s], lambda x: x,
             )
+
+    def test_no_query_on_parent_access(self):
+        # Refs #15250
+        place1 = Place(
+            name="Guido's House of Pasta",
+            address='944 W. Fullerton')
+        place1.save_base(raw=True)
+        restaurant = Restaurant(
+            place_ptr=place1,
+            serves_hot_dogs=True,
+            serves_pizza=False)
+        restaurant.save_base(raw=True)
+        italian_restaurant = ItalianRestaurant(
```

Comment:
can we create the `ItalianRestaurant` and then get the related entites we need from that? I think that would be a bit nicer than `save_base(raw=True)`.